x/data/msg_anchor.go

```diff
@@ -3,7 +3,7 @@ package data
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-	"github.com/cosmos/cosmos-sdk/x/auth/legacy/legacytx"
+	"github.com/cosmos/cosmos-sdk/x/auth/migrations/legacytx"
```

Comment:
Ok, to leave for now but wondering if we need to still support legacy messages...?

Reply:
Yes, for ledger signing.